studio/pages/project/[ref]/settings/database.tsx

```diff
@@ -303,14 +303,18 @@ const GeneralSettings: FC<any> = ({ projectRef }) => {
   const DB_FIELDS = ['db_host', 'db_name', 'db_port', 'db_user', 'inserted_at']
   const connectionInfo = pluckObjectFields(formModel, DB_FIELDS)
 
-  const defaultConnString =
+  const defaultUriConnString =
     `postgresql://${connectionInfo.db_user}:[YOUR-PASSWORD]@` +
     `${connectionInfo.db_host}:${connectionInfo.db_port.toString()}` +
     `/${connectionInfo.db_name}`
   const golangConnString =
     `user=${connectionInfo.db_user} password=[YOUR-PASSWORD] ` +
     `host=${connectionInfo.db_host} port=${connectionInfo.db_port.toString()}` +
     ` dbname=${connectionInfo.db_name}`
+  const defaultPsqlConnString = 
+    `psql -h db.${connectionInfo.db_host}.supabase.co -p ` +
```

Comment:
`.supabase.co` should not be present. `db.` as well.

Should interpolate the `connectionInfo.db_host` directly, see the uri version